src/sst/core/statapi/statgroup.cc

```diff
@@ -50,7 +50,7 @@ bool
 StatisticGroup::containsStatistic(const StatisticBase* stat) const
 {
     if ( isDefault ) return true;
-    std::find(stats.begin(), stats.end(), stat);
+    std::ignore = std::find(stats.begin(), stats.end(), stat);
```

Comment:
I assume this was added to squash a warning, but the warning actually revealed a bug.  The code should check the return value of the find and return the proper bool.  So, something like this:

if ( std::find(stats.begin(), stats.end(), stat) != stats.end() ) return true;

We need to fix this instead of suppressing the warning.